src/App.tsx

```diff
@@ -1,15 +1,28 @@
-import React from 'react';
 import 'bulma/bulma.sass';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
-import classNames from 'classnames';
-import { PostsList } from './components/PostsList';
-import { PostDetails } from './components/PostDetails';
-import { UserSelector } from './components/UserSelector';
+import cn from 'classnames';
+import { useEffect } from 'react';
+import { PostsList } from './components/PostsList/PostsList';
+import { PostDetails } from './components/PostDetails/PostDetails';
+import { UserSelector } from './components/UserSelector/UserSelector';
 import { Loader } from './components/Loader';
+import { usePosts, useUsers } from './context';
 
 export const App: React.FC = () => {
+  const { selectedUser } = useUsers();
+  // Prettier does it)))
+  /* eslint-disable */
```

Comment:
let's try without eslint disable)

Reply:
> let's try without eslint disable)

I have to disable Prretier rules to do that I think it's not worth it.